### DISCUSSION TITLE
duplicate or create flux component

### DISCUSSION THREAD

**Discussion by vincentcordel** [livewire/flux#525] [Feature Requests]
it’d be nice to be able to be able to duplicate a component and/or create a new one using the same flux syntax instead of using the default livewire syntax (to be more consistent)

Maybe it’s already possible then how?

**Comment by joshhanley** [comment]
@vincentcordel I don't follow what you mean. Can you provide a better description with some code examples? Thanks!

**Comment by vincentcordel** [comment]
1st case (works):
Let's say I want to duplicate the flux heading component.
I created heading2 with different styling. This works as expected with `<flux:heading2 ... />`

2nd case (don't know if it's possible):
I'm using a custom component for a date picker. Currently in app/View/Components/DateTime.php as an inline component (`Render <<<<HTML

which I render in my view as `<x-datetime .../>`

What I'd like to achieve is to convert it to a flux component so I can use `<flux:datetime ... />`

It'd also be useful for other components that nest flux components.
Ex. a form container or a repeater
```
<x:flux:formcontainer>
  <x:input ... />
  ...
</x:flux:formcontainer>
```

**Reply by joshhanley** [reply]
@vincentcordel I think if you put your custom components (ie date) inside the Flux published folder, then it should work (as `<flux:date>`). But it would need to be an anonymous component, not class backed I'd think.
